@xen-orchestra/lite/src/components/AppHeader.vue

```diff
@@ -22,15 +27,75 @@ import PoolOverrideWarning from '@/components/PoolOverrideWarning.vue'
 import TextLogo from '@/components/TextLogo.vue'
 import UiIcon from '@/components/ui/icon/UiIcon.vue'
 import { useNavigationStore } from '@/stores/navigation.store'
+import { usePoolCollection } from '@/stores/xen-api/pool.store'
 import UiButton from '@core/components/button/UiButton.vue'
 import { useUiStore } from '@core/stores/ui.store'
-import { faBars, faDownload } from '@fortawesome/free-solid-svg-icons'
+import { faBars, faDownload, faArrowUpRightFromSquare } from '@fortawesome/free-solid-svg-icons'
 import { storeToRefs } from 'pinia'
+import { computed } from 'vue'
 import { useRouter } from 'vue-router'
+import type { PRIMARY_ADDRESS_TYPE } from '@/libs/xen-api/xen-api.enums'
 
 const router = useRouter()
+const { pool } = usePoolCollection()
```

Comment:
Since this [PR](https://github.com/vatesfr/xen-orchestra/pull/7602) is merged, you can rebase and use the new version to subscribe to the store:
```suggestion
const { pool } = usePoolStore().subscribe()
```